scripts.js

```diff
@@ -12,3 +12,20 @@ function hide() {
 
 hamburger.addEventListener('click', show);
 closeBtn.addEventListener('click', hide);
+
+// Form validation
+const userEmail = document.querySelector('#email');
+const alertMsg = document.querySelector('.alert-Msg');
+const contactForm = document.querySelector('.input-Form');
+
+contactForm.addEventListener('submit', (event) => {
+  const regex = /^([a-z\d-]+)@([a-z\d-]+)\.([a-z]{2,8})(\.[a-z]{2,8})?$/;
+  const emailContent = userEmail.value;
+  if (regex.test(emailContent)) {
+    alertMsg.innerHTML = 'Success';
+    alertMsg.style.color = 'green';
+  } else {
+    alertMsg.innerHTML = 'Email address should all be lower case';
+    event.preventDefault();
+  }
```

Comment:
- [ **OPTIONAL** ] **IMPORTANT:** Good job displaying your error message close to the **submit** button. But I think it would be better you display it between the `textarea` field and the `Submit` button, so even if the didn't scroll the entire page till the bottom, he will sill see the error message. It can also be good to change the text color to `red` as it shows an error message